Jenkinsfile.nightly

```diff
@@ -32,17 +32,16 @@ node ('macos1'){
 
       checkout scm
 
-      latest_tag = sh(returnStdout: true, script: 'git describe --tags `git rev-list --tags --max-count=1`').trim()
-
       sh 'git fetch --tags'
-      sh 'git tag -D ' + latest_tag
+      latest_tag = sh(returnStdout: true, script: 'git describe --tags `git rev-list --tags --max-count=1`').trim()
+      sh 'git tag -d ' + latest_tag
       sh 'rm -rf node_modules'
       sh 'cp .env.jenkins .env'
       sh 'lein deps'
       installJSDeps()
 
       sh 'mvn -f modules/react-native-status/ios/RCTStatus dependency:unpack'
-        sh 'cd ios && pod install && cd ..'
+        sh 'cd ios &&  pod install && cd ..'
```

Comment:
does the extra space change anything here ?

Reply:
i doubt :D